libandroid-navigation/src/test/java/com/mapbox/services/android/navigation/v5/navigation/MapboxNavigationTest.java

```diff
@@ -70,6 +69,39 @@ public void defaultEngines_didGetInitialized() throws Exception {
     assertNotNull(navigation.getOffRouteEngine());
   }
 
+  @Test
+  public void offRouteEngine_doesNotGetInitializedWithFalseOption() throws Exception {
```

Comment:
Normally is good to structure the tests following the triple A 👀 https://speakerdeck.com/guardiola31337/elegant-unit-testing-mobilization-2016?slide=40 http://wiki.c2.com/?ArrangeActAssert
If the `A`rrange part gets too long we should create factory or builder methods.

This applies to other tests included in `MapboxNavigationTest`.